zio-flow/shared/src/main/scala/zio/flow/operation/http/package.scala

```diff
@@ -0,0 +1,43 @@
+package zio.flow.operation
+
+import zio.json.{JsonCodec, JsonDecoder}
+import zio.flow.operation.http.API.NotUnit
+import zio.json.internal.{RetractReader, Write}
+import zio.schema.Schema
+
+import java.util.UUID
+import scala.language.implicitConversions
+
+package object http {
+
+  // Paths
+  val string: Path[String]   = Path.Match("string", Schema[String])
```

Comment:
It would be nice to be able to have a `contramap` for these `Match` and `SingleParam` fragments so we can directly use domain specific newtypes